telosnetwork/telos-wallet
# send page visual mockup

# Fixes #174

## Description
We need a visual component for the send page, following the new wallet design. This PR includes that component and implements some minor buttons like clicking on the Available amount or selecting a token from a hard-coded list.

## Test scenarios
- go to [this link](https://deploy-preview-232--wallet-staging.netlify.app/)
- login with Metamask
- press the Send button on the page header
  - you should see a send form like the screenshot shows
- press the available amount link on top of the input
  - the available amount text should be copied to the input
- press the cancel btn
  - you should be redirected to the previous page

## Screenshots
![image](https://user-images.githubusercontent.com/4420760/230285326-9cf28bd8-2362-4376-bbe3-7da3a98f7586.png)




## Checklist:
